### ISSUE TITLE
site not opening. 

### ISSUE THREAD

**Issue by machulnyy** [bradp/vv#319]
Hi all,

I followed the install instructions and what seemed like successfully created a new site using 

`vv create`

The site folder appeared in my specified directory. However it only contained SiteName.conf, vvv-hosts, vvv-init.sh, wp-cli.yml files. So when I went onto the VVV dashboard, I could see the site listed but without option to visit the site or the Wordpress admin. After doing some searching around I found the solution was to run 

`vagrant provision`
in the site directory.

this successfully created an htdocs folder with all the Wordpress files. 

Now when I go onto the VVV dashboard, the options to visit the site, admin, profiles, themes etc, are all there, but when I press visit or admin, nothing happens. 

Also, when I try to access the site directly via: SiteName.dev, it opens the VVV dashboard.

Am I missing something?

And if this is anything to go by, the last time I did `vagrant provision`, this was one of the several RED lines of code in the terminal when the process was running `==> default: Error: Can’t select database. We were able to connect to the database server (which means your username and password is okay) but not able to select the`SiteName`database.
`

Thank you for any input!

**Comment by machulnyy** [comment]
Can anyone help me out please?

**Comment by vlood** [comment]
I got the very same issue after running `vagrant up --provision`. Something with nginx configuration must be wrong, having the domain being redirected to the dashboard again.

**Comment by metaColin** [comment]
Having the same problem. No amount of 'vagrant destroy' or 'vagrant provision' will make a difference.

**Comment by jarvisjohnson** [comment]
Same here.  Tried with an older version on VVV and VV.  Updated VVV to today's release, updated VV with `VV --update`, still same thing.  Site goes to VVV dashboard.  Can't find the database in PHPmyadmin either.

**Comment by jarvisjohnson** [comment]
I did notice that when updating VVV, there was one console error for a connection to the wordpress.org server.  Perhaps something to do with it?

**Comment by benlumia007** [comment]
Please see ticket #324

**Comment by sthag** [comment]
Hi,

I also ran in this issue after testing vv for the first time. @benlumia007 I think the problems described in the ticket #324 are happing here also but they are not necessarily the same.
I'm on Windows 10, vv is 1.12, vvv is >1.4.0, vagrant is 1.8.7 and VirtualBox is 5.1.2.

I noticed two things. After running `vv create` everything seems to be going fine but wordpress is not installed. This is described here #324 . A workaround seems to be to start `./vvv-init.sh` manually like @sdulaney described. This installed wordpress but brought me here to this issue because it is installed into an `htdocs` directory inside the named directrory during the `vv create` process.
This does not seem to be correct. VVV itself installs the wordpress files directly into the newsite directory. There is no directory called `htdocs` created. So it looks like this:

- wordpress-default
  - wp-files
- wordpress-develop
  - wp-files
- vv-create-directory
  - htdocs
    - wp-files

I guess this is the reason why the domain, which is correctly added, leads to the dashboard. Even if I call `http://{domainname}/htdocs/` it's showing the dashboard.
Also the database maybe is not in a default place. Looking into phpmyadmin all the default vvv databases are bundled inside a container called `wordpress`. The new one from `vv create` is placed outside the container.
